### PR TITLE
ALP: isolated interactive Agama deployment

### PR DESCRIPTION
**Pull request by tbazant**
### Description

In parallel with the existing ALP guide, this is an attempt to isolate one specific deployment method.

### CODE REVIEW

File: references/alp-requirements.xml [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+  </info>
+  <section xml:id="alp-deployment-requirements-hardware">
+    <title>Hardware requirements</title>
+    <variablelist>
+      <varlistentry>
+        <term>BIOS</term>
+        <listitem>
+          <para>
+            Installing &productnameshort; is supported on hosts with UEFI BIOS
+            only. Hosts with the legacy BIOS are not supported.
```
Comment (tomschr) [inline references/alp-requirements.xml:31]:
```suggestion
            only. Hosts with legacy BIOS are not supported.
```

File: references/alp-requirements.xml [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+    <meta name="maintainer" content="tbazant@suse.com" its:translate="no"/>
+    <abstract>
+      <para>
+        This article lists requirements for deploying &productnameshort;.
+      </para>
+    </abstract>
+  </info>
+  <section xml:id="alp-deployment-requirements-hardware">
+    <title>Hardware requirements</title>
+    <variablelist>
```
Comment (tomschr) [inline references/alp-requirements.xml:25]: In the section "Virtualized deployment" you have a paragraph to introduce it. For consistency reasons, maybe do the same here? I had this in mind:

```suggestion
    <para>
       When deploying &productnameshort; on bare-metal or as a virtual machine
       guest, make sure the following parameters are met:
    </para>
    <variablelist>
```

File: tasks/prepare-alp-vm.xml [hunk 3]
```diff
@@ -21,232 +21,188 @@
...
+          Btrfs file system to the desired size, for example:
         </para>
+<screen>&prompt.root;btrfs fi resize max /</screen>
       </step>
       <step>
         <para>
-          If you are deploying an encrypted &productnameshort; image, perform
-          these additional steps:
+          Leave the shell with <command>exit</command> and reboot the system
+          with <command>reboot</command>.
```
Comment (tomschr) [inline tasks/prepare-alp-vm.xml:73]: Isn't there a "now" missing?

```suggestion
          with <command>reboot now</command>.
```

File: tasks/prepare-alp-vm.xml [hunk 4]
```diff
@@ -21,232 +21,188 @@
...
+  </important>
+  <procedure>
+    <step>
+      <para condition="alp-deployment-all">
+        Depending on the deployment method, download either the &alp-installer;
+        image or the raw disk image of &productnameshort; from
+        <link xlink:href="https://susealp.io/downloads/"/> on the &vmhost;
+        where you intend to run virtualized &productnameshort;.
+      </para>
+      <para condition="alp-deployment-agama-interactive;alp-deployment-rawdisk">
```
Comment (tomschr) [inline tasks/prepare-alp-vm.xml:86]: If this is for both variants, you don't need the `condition` attribute:

```suggestion
      <para>
```

**Review by tomschr — COMMENTED** [review]
@tbazant Very impressive! :+1:
I have some formal questions/suggestions. Maybe it's helpful. :slightly_smiling_face:

File: articles/deploying-alp-interactively-using-agama.asm.xml
[the remark this answers is not in the document]
Comment (fsundermeyer) [reply articles/deploying-alp-interactively-using-agama.asm.xml:1]: Yes, it works. You need to set STRUCTID=<ID> in the DC file, that's all

**Pull request merged**